src/abi/receipts.md

```diff
@@ -124,6 +126,46 @@ _Important note:_ For the JSON representation of receipts, we represent 64-bit u
 }
 ```
 
+## Mint Receipt
+
+- `type`: `Mint`.
+- `sub_id`: Hexadecimal string representation of the 256-bit (32-byte) asset sub identifier ID.
+- `contract_id`: Hexadecimal string representation of the 256-bit (32-byte) contract ID of the current context.
+- `val`: Decimal string representation of a 64-bit unsigned integer; value of register `$rA`.
+- `pc`: Hexadecimal string representation of a 64-bit unsigned integer; value of register `$pc`.
+- `is`: Hexadecimal string representation of a 64-bit unsigned integer; value of register `$is`.
+
+```json
+{
+  "type": "Mint",
+  "sub_id": "0x39150017c9e38e5e280432d546fae345d6ce6d8fe4710162c2e3a95a6faff051",
+  "contract_id": "0x39150017c9e38e5e280432d546fae345d6ce6d8fe4710162c2e3a95a6faff051",
```

Comment:
this breaks our convention of using `id` for the contract like other receipts. Any particular reason we used contract_id instead of just `id`? cc @SilentCicero @dmihal